src/main/kotlin/no/nav/syfo/dialogmotekandidat/DialogmotekandidatService.kt

```diff
@@ -27,8 +27,8 @@ class DialogmotekandidatService(
     fun getLatestOppfolgingstilfelle(personIdentNumber: PersonIdentNumber): OppfolgingstilfelleArbeidstaker? =
         oppfolgingstilfelleService.getSisteOppfolgingstilfelle(personIdentNumber)
 
-    fun getDialogmotekandidatMedStoppunktPlanlagtTodayList(): List<DialogmotekandidatStoppunkt> =
-        database.getDialogmotekandidatStoppunktTodayList().toDialogmotekandidatStoppunktList()
+    fun getDialogmotekandidatMedStoppunktPlanlagtTodayOrYesterdayList(): List<DialogmotekandidatStoppunkt> =
```

Comment:
`med` -> `with`?
Og er det bedre med `getDialogmotekandidaterMedStoppunktTodayOrYesterday` med kandidater i flertall, i stedet for å ha med `List` i metodenavnet?